client/asset/dcr/dcr.go

```diff
@@ -1005,10 +1024,34 @@ func (dcr *ExchangeWallet) convertCoin(coin asset.Coin) (*output, error) {
 	return newOutput(dcr.node, txHash, vout, coin.Value(), tree, coin.Redeem()), nil
 }
 
+func (dcr *ExchangeWallet) sendToAddress(addr dcrutil.Address, val uint64, nfo *dex.Asset) (*wire.MsgTx, error) {
```

Comment:
I was initially using `(*rpcclient.Client).SendToAddress`, but ran into a problem. `SendToAddress` returns a transaction hash. We need to know the index of the relevant output. Fetching the transaction details with `GetTransaction` should be fine, but the `Amount` field of the resulting `GetTransactionDetailsResult` struct is oddly negative sometimes. Negative value verified using dcrctl.